mods/ra/chrome/ingame-observer.yaml

```diff
@@ -481,6 +482,42 @@ Container@OBSERVER_WIDGETS:
 									Font: Bold
 									Text: Support Powers
 									Shadow: True
+						Container@ARMY_HEADERS:
+							X: 0
+							Y: 0
+							Width: 400
+							Height: PARENT_BOTTOM
+							Children:
+								ColorBlock@HEADER_COLOR:
+									X: 0
+									Y: 0
+									Color: 00000090
+									Width: PARENT_RIGHT - 200
+									Height: PARENT_BOTTOM
+								GradientColorBlock@HEADER_GRADIENT:
+									X: PARENT_RIGHT - 200
+									Y: 0
+									TopLeftColor: 00000090
+									BottomLeftColor: 00000090
+									Width: 200
+									Height: PARENT_BOTTOM
+								Label@PLAYER_HEADER:
+									X: 40
```

Comment:
This ends up being a few pixels too far to the right for some reason - only in d2k though...?

Reply:
this is the file for RA?